WebSocket/homePage.go

```diff
@@ -0,0 +1,13 @@
+package WebSocket
+
```

Comment:
same, this file is the same as the one provided in the example of Gorilla / Websocket.